tensorflow/python/keras/applications/vgg16.py

```diff
@@ -87,6 +87,15 @@ def VGG16(include_top=True,
   Raises:
       ValueError: in case of invalid argument for `weights`,
           or invalid input shape.
+          
+          
+  Usage Example:
+  ```python
```

Comment:
Hello, to make the doctests work, you need to remove these " \`\`\`python " and " \`\`\` " at the end of the code.

Reply:
So,the usage example code should still be in the comments?

Reply:
I don't understand what you mean by "should code still be in comments" but I forgot to mention that the code lines should start with `>>>` instead of `>>` to make the doctests work. And could you please remove blank lines that you put when you deleted the " \`\`\`python " and " \`\`\` " ?

Reply:
All done. Is it ok now?